package android.graphics;

/* JADX INFO: loaded from: classes.dex */
public class DiscretePathEffect extends PathEffect {
    private static native int nativeCreate(float f, float f2);

    public DiscretePathEffect(float f, float f2) {
        this.native_instance = nativeCreate(f, f2);
    }
}
